execution_engine/src/core/runtime/mod.rs

```diff
@@ -2819,6 +2819,41 @@ where
         Ok(Ok(()))
     }
 
+    /// Reads the `value` under a `key` in a dictionary.
+    fn dictionary_read(
```

Comment:
move this up next to dictionary_get

it should also be possible to dovetail the internals of get / read as they are just two different ways of identifying the same record

Reply:
Moved and addressed in https://github.com/casper-network/casper-node/pull/3161/commits/862d2b060f8391957da23d76025c2b8a6fd92c5f